### PR TITLE
Mention new-cap rule

### PR DESCRIPTION
**Pull request by ender503**
Make a code example to describe the new-cap rule in the style guide.

The new-cap rule:
The function name starts with the upper case which means constructor,
otherwise regular function.

Follow the link to know more detail about `new-cap` rule:
 - https://eslint.org/docs/rules/new-cap
 - https://github.com/airbnb/javascript#naming--PascalCase

### CODE REVIEW

File: STYLE-GUIDE.md [hunk 1]
```diff
@@ -5,6 +5,39 @@
...
 with the following exceptions or extra manner.
 
+### Require constructor names to begin with a capital letter
+
+Since the constructor functions are just regular functions, it is
+recommended to use thse uppercase to recognize which functions are to be
+used as constructors.
+
+```javascript
+
```
Comment (jserv) [inline STYLE-GUIDE.md]: No extra blank line here.

File: STYLE-GUIDE.md [hunk 2]
```diff
@@ -5,6 +5,39 @@
 TangleID defers to [Airbnb JavaScript Style Guide](https://github.com/airbnb/javascript)
 with the following exceptions or extra manner.
 
+### Require constructor names to begin with a capital letter
+
+Since the constructor functions are just regular functions, it is
+recommended to use thse uppercase to recognize which functions are to be
```
Comment (jserv) [inline STYLE-GUIDE.md]: Change "to recognize..." to "specify constructors".

**Comment by jserv** [comment]
Can eslint manage to fix such violations?

**Comment by ender503** [comment]
ESLint only detects this violation.

**Pull request merged**